semantic.pixelator/src/app/ui.R

```diff
@@ -15,18 +15,24 @@ semanticPage(
     header = div(
       class = "ui raised segment inverted",
       div(class = "background-gradient"),
+      tags$a(
+        class = "header-logo",
+        href = "https://appsilon.com/",
+        target = "_blank",
+        tags$img(src = "assets/appsilon-logo-mark.png")
+      ),
       h2(class = "title", "Semantic Pixelator")
     ),
 
     side = div(
       class = "sidebar-wrapper",
       gridPanel(class = "ui raised segment inverted sidebar-container",
         gap = "15px",
-        rows = "35px 322px 1fr 40px 100px",
+        rows = "70px 322px 1fr 40px 100px",
 
         div(
           id = "image-picker-buttons",
-          darkify(fileInput, "upload", label = "", buttonLabel = "Upload image", type = "file", accept = "image/png, image/jpeg"),
+          fileInput("upload", label = "", buttonLabel = "Upload image", type = "file", accept = "image/png, image/jpeg"),
```

Comment:
can we add some sort of check to test if the the user is trying to upload an accepted file extension?